Extensions/Security/Authentication.md

```diff
@@ -0,0 +1,35 @@
+# Authentication Extension
+
+_This extension specification is currently incubating.  While incubating the version is 0._
+
+## Introduction
+Authentication is a necessary component to any real world application. This extension specification provides a standardized mechanism for including both the type of credentials and the credentials in metadata payloads.
+
+## Metadata Payload
+This metadata type can be used in a per connection or per stream, and not individual payloads and as such it **MUST** only be used in frame types used to initiate interactions and payloads.  This includes [`SETUP`][s], [`REQUEST_FNF`][rf], [`REQUEST_RESPONSE`][rr], [`REQUEST_STREAM`][rs], [`REQUEST_CHANNEL`][rc], and [`PAYLOAD`][p].  The Metadata MIME Type is `message/x.rsocket.authentication.v0`.
```

Comment:
Looks like it is a common part of every extension definition -> for instance https://github.com/rsocket/rsocket/blob/master/Extensions/Routing.md#metadata-payload

Reply:
@yschimke You are right that we should not list `PAYLOAD` here because it is not a start of a stream or connection. I pushed https://github.com/rsocket/rsocket/pull/275/commits/fa1a80b9212816b2efce0c538d06ba5366d1a32b to address this.

I kept the commit separate, so it was easier to see the changes. I can clean up the commits later if desired.